tests/feature_activation/test_feature_service.py

```diff
@@ -43,8 +43,8 @@ def _get_blocks_and_storage() -> tuple[list[Block], TransactionStorage]:
         0b0011,
         0b0001,
 
-        0b0000,  # 8: boundary block
-        0b0000,
+        0b0010,  # 8: boundary block
```

Comment:
Are these bits activated because of MUST_SIGNAL?

Reply:
Yes, those specific bits in those block heights were not being used by any other test in this file, so they could be freely changed. I simply configured the new `MUST_SIGNAL` tests to use them.

Reply:
Got it. Thanks!